app/controllers/coronavirus_form/nhs_number_controller.rb

```diff
@@ -18,9 +18,11 @@ def submit
         format.html { render controller_path, status: :unprocessable_entity }
       end
     elsif session[:check_answers_seen]
+      session[:know_nhs_number] = I18n.t("coronavirus_form.questions.know_nhs_number.options.option_yes.label")
```

Comment:
Move this up the block so you only have to have this line in there once.

Thanks for adding a test for this!

Reply:
Fixed - thanks @bilbof